docs/rooms/exampleRoom3Room.md

```diff
@@ -18,21 +18,21 @@ This room will go away when other rooms are implemented.
 
 ## Commands
 
-- **leave**: You leave example room 3
-- **go3A**: You move to state 3A
-- **go3B**: You move to state 3B
-- **goStart**: You move back to the entrance
+- **leave**: You leave example room 3 having completed n laps
```

Comment:
I've been using `X` for vars. Example: https://github.com/normal-devs/klicker-knight/blob/main/docs/rooms/fishingRoom.md?plain=1#L7

Should we update this to `n`?

Reply:
I don't see a need to make the variables consistent between docs. I just think we need to be consistent within docs. I assume at some point we will have rooms with multiple variables and IMO we should pick whatever references we think make sense for that room.